Failing to build docs with Docker

## What happened?
`docs` container exited with code 2



## What should've happened instead?
`docs` container is UP and I can be able to use it



## Steps to reproduce
1. Make new project
2. `docker-compose -f local.yml build`
3. `docker-compose -f local.yml up docs`

`Dockerfile`
```
FROM python:3.8-slim-buster

ENV PYTHONUNBUFFERED 1
ENV PYTHONDONTWRITEBYTECODE 1

RUN apt-get update \
    # dependencies for building Python packages
    && apt-get install -y build-essential \
    # psycopg2 dependencies
    && apt-get install -y libpq-dev \
    # Translations dependencies
    && apt-get install -y gettext \
    # Uncomment below lines to enable Sphinx output to latex and pdf
    # && apt-get install -y texlive-latex-recommended \
    # && apt-get install -y texlive-fonts-recommended \
    # && apt-get install -y texlive-latex-extra \
    # && apt-get install -y latexmk \
    # cleaning up unused files
    && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
    && rm -rf /var/lib/apt/lists/*

# Requirements are installed here to ensure they will be cached.
COPY ./requirements /requirements
# All imports needed for autodoc.
#RUN pip install -r /requirements/local.txt -r /requirements/production.txt
RUN pip install -r /requirements/requirements.txt

WORKDIR /docs

CMD make livehtml

```

`local.yml`
```
...
  docs:
    image: frame_and_gap_local_docs
    container_name: docs
    build:
      context: .
      dockerfile: ./compose/local/docs/Dockerfile
    env_file:
      - ./.envs/.local/.django
    volumes:
      - ./docs:/docs
      - ./config:/app/config
      - ./frame_and_gap:/app/frame_and_gap
    ports:
      - "7000:7000"
```

Then `docker-compose -f local.yml up docs`
```bash
(frame_and_gap) KrungThep:frame_and_gap sarit$ docker-compose -f local.yml up docs
Creating network "frame_and_gap_default" with the default driver
Creating docs ... done
Attaching to docs
docs            | sphinx-autobuild -b html -H 0.0.0.0 -p 7000 --watch /app -c . ./_source ./_build/html
docs            | usage: sphinx-autobuild [-h] [--port PORT] [--host HOST]
docs            |                         [--re-ignore RE_IGNORE] [--ignore IGNORE]
docs            |                         [--no-initial] [--open-browser] [--delay DELAY]
docs            |                         [--watch DIR] [--pre-build COMMAND] [--version]
docs            |                         sourcedir outdir [filenames [filenames ...]]
docs            | sphinx-autobuild: error: unrecognized arguments: -H -p ./_source ./_build/html
docs            | make: *** [Makefile:20: livehtml] Error 2
docs exited with code 2
```

--- a/{{cookiecutter.project_slug}}/docs/Makefile
+++ b/{{cookiecutter.project_slug}}/docs/Makefile
@@ -22,9 +22,9 @@ help:
 # Build, watch and serve docs with live reload
 livehtml:
 	sphinx-autobuild -b html 
-	{%- if cookiecutter.use_docker == 'y' %} -H 0.0.0.0 
+	{%- if cookiecutter.use_docker == 'y' %} --host 0.0.0.0
 	{%- else %} --open-browser 
-	{%- endif %} -p 7000 --watch $(APP) -c . $(SOURCEDIR) $(BUILDDIR)/html
+	{%- endif %} --port 7000 --watch $(APP) -c . $(SOURCEDIR) $(BUILDDIR)/html
 
 # Outputs rst files from django application code
 apidocs:

